crates/containerd-shim-wasm/src/container/context.rs

```diff
@@ -36,6 +36,16 @@ pub trait RuntimeContext {
     // the platform for the container using the struct defined on the OCI spec definition
     // https://github.com/opencontainers/image-spec/blob/v1.1.0-rc5/image-index.md
     fn platform(&self) -> &Platform;
+
+    // the container id for the running container
+    fn container_id(&self) -> &str;
+
+    // the pod id for the running container (if available)
+    // In Kubernetes environments, containers run within pods, and the pod ID is usually
+    // stored in the OCI spec annotations under "io.kubernetes.cri.sandbox-id"
+    fn pod_id(&self) -> Option<&str> {
+        None
+    }
```

Comment:
I think this is fine.  We might consider putting this behind a struct/function that hides this behind something like `info()` if we have more things we might want to log.  But as of now I can't think of anything.